app/views/feedback/bug_report.html.haml

```diff
@@ -1,39 +1,32 @@
 = content_for :page_title, flush: true do
   = t('.page_title')
 
-= render partial: 'errors/error_summary', locals: { ep: @feedback, error_summary: t('.error_summary') }
 = render partial: 'layouts/header', locals: { page_heading: t('.page_heading') }
 
-.grid-row
-  .column-two-thirds
-    = govuk_inset_text do
-      %p
-        = t('.intro')
-        = render partial: 'feedback/claim_edit_alert' if referrer_is_claim?(@feedback.referrer)
-
-    = form_for @feedback, builder: AdpFormBuilder, url: feedback_index_path, html: { novalidate: 'novalidate', class: 'normal'} do |f|
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    = form_for @feedback, builder: GOVUKDesignSystemFormBuilder::FormBuilder, url: feedback_index_path, html: { novalidate: 'novalidate' } do |f|
       = f.hidden_field :type
       = f.hidden_field :referrer
 
-      .form-group
-        = f.label :case_number, class: 'form-label', id: :case_number do
-          = t('.case_number_label')
-          %span.form-hint
-            = t('.case_number_hint')
-        = f.text_field :case_number, class: 'form-control'
+      = f.govuk_error_summary
```

Comment:
does this work with links to fields as expected? great if it does. I suspect it won't on the more complex forms

Reply:
asking because I am not sure how much test coverage there is for the unhappy paths.